kuksa-appstore/src/main/java/org/eclipse/kuksa/appstore/utils/Utils.java

```diff
@@ -106,4 +118,21 @@ public static boolean isUserAlreadyOwner(User user,
 		}
 		return ownerList;
 	}
+
+	public static List<String> convertPermissionArtifactFileToStringList(String permissionArtifactString)
```

Comment:
For the name of the method is a bit confusing because it contains "File" while the method actually has a String input. What about "permissionArtifactStringToList" as name?

Reply:
I will replace it. OK.

Reply:
Thank you for the quick change.